src/libs/actions/Report.ts

```diff
@@ -858,8 +870,9 @@ function openReport(
     };
 
     const isInviteOnboardingComplete = introSelected?.isInviteOnboardingComplete ?? false;
+    const isOnboardingCompleted = onboarding?.hasCompletedGuidedSetupFlow ?? false;
```

Comment:
How's this looking @blazejkustra?

Reply:
I was testing for an olddot user, sorry for the delay!

Reply:
![image](https://github.com/user-attachments/assets/650a6376-a084-4575-8f67-9cbc5c968e20)

This is not an issue because `introSelected` NVP is not set for users from olddot, therefore this if is always falsy:
```
if (introSelected && !isOnboardingCompleted && !isInviteOnboardingComplete) {
```